packages/jellyfish-api-core/__tests__/category/loan/closeVault.test.ts

```diff
@@ -34,7 +35,23 @@ describe('Loan', () => {
     const collateralAddress = await tGroup.get(0).container.getNewAddress()
     await tGroup.get(0).token.dfi({ address: collateralAddress, amount: 200000 })
 
-    // oracle setup
+    const loanTokenMinterProvider = tGroup.get(1)
```

Comment:
feel bad about having the `Minter` keyword in this. Coz we are not actually minting loan tokens.
How about we change the `loanTokenMinterProvider` to `loanTokenProvider` and `loanTokenMinterAddr` to `loanTokenProviderAddr`